lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -530,9 +568,14 @@ def get_array_section_has_problem(course_id):
         for subsection in section.get_children():
             for unit in subsection.get_children():
                 for child in unit.get_children():
-                    if child.location.category == 'problem':
+                    if child.location.category in PROB_TYPE_LIST:
```

Comment:
PROB_TYPE_LIST includes 'lti', is this intended? Was the old check intentionally excluding 'lti' type problems?